src/main/java/org/fungover/haze/Main.java

```diff
@@ -80,7 +80,7 @@ private static void controlCommand(HazeList hazeList, HazeDatabase hazeDatabase,
         inputList.forEach(System.out::println); // For checking incoming message
     }
 
-    private static List<String> getInputList(BufferedReader input) throws IOException {
```

Comment:
Any special reason for changing into public methods instead of private?

Reply:
no, we where trying some things and forgot to change it back,